src/ansible/roles/ssh_server/tasks/main.yml

```diff
@@ -11,3 +11,7 @@
     name: '{{ systemd.services.sshd }}'
     enabled: yes
     state: restarted
+  register: restart_res
```

Comment:
What issues did you see? I never run into any issue when provisioning this in downstream ci.

Reply:
I had a multiple completely random failures there probably due to openstack being under bigger load.